app/src/gui/components/notebook/delete.tsx

```diff
@@ -172,13 +161,9 @@ export default function RecordDelete(props: RecordDeleteProps) {
       >
         <Alert severity="error">
           <AlertTitle>
-            Are you sure you want to {is_draft ? 'discard draft' : 'record'}{' '}
-            {is_draft ? draft_id : record_id}?
+            Are you sure you want to delete record {recordId}?
           </AlertTitle>
-          You cannot reverse this action! Be sure you wish to proceed
-          {!is_draft
-            ? ' (all associated drafts of this record will also be removed).'
-            : '.'}
+          You cannot reverse this action! Be sure you wish to proceed.
```

Comment:
Ah, but it can be reversed. We should have some UI for viewing deleted records and recovering them.